Sprint_Tests/maria_ptitsyna.md

```diff
@@ -0,0 +1,12 @@
+1. He used to manage well in layout and work with actual technologies in front end. After small break in several years and major depression plus listening hard music, he weakly manages in actual tendentious and drinks a lot. If he had overcome the depression after dead of his wife and child, he wouldn't have had a question about search of a job with a relevant salary.
+
```

Comment:
manage is used when you want to say that somebody coped with somthing in time in our case the verb BE is enough or any other variation.
He used to be really good in markup
top notch technologies
After a small break for/of several years
listening to hard music
he is not much knowlegable about modern/contemporary tendencies
after his wife's and child's death
he wouldn't have an issue with looking for a job with the relevant salary
(mixed conditional in the last sentence)